docs/guides/role.md

```diff
@@ -1,19 +1,54 @@
 # Roles
-[Role Interface](../api/interfaces/cacheServerClient_cacheServerClient_types.IRole.md)  
 
-/*
-TO DO: 
-//Define Role in context of iam-client-lib/Switchboard
+[Role Interface](../api/interfaces/cacheServerClient_cacheServerClient_types.IRole.md)
 
-//Give an example(s) of a Role
-*/
+**Role** is leaf in EnergyWeb domains tree. Its main purpose is to distribute responsibilities
+and permissions in some domain. As part of its definition, role determines the order of its
+issuance and revoсation. In particular requesting of role can require preliminary assignment
+of some other role. The similar applies to issuance. Role can be placed under organization or
+application thus defining its scope. Organization role serves organization government purposes,
+while application roles mainly describes application clients.
```

Comment:
I would remove this bit. I'm not sure what "government purposes" refers to here.
```suggestion
application thus defining its scope.
```